docs/pages/modules/get-started.mdx

```diff
@@ -4,19 +4,128 @@ sidebar_title: Get started
 description: Learn about getting started with Expo modules API.
 ---
 
+import { BookOpen02Icon, Grid01Icon } from '@expo/styleguide-icons';
+
 import { BoxLink } from '~/ui/components/BoxLink';
+import { Step } from '~/ui/components/Step';
 import { Terminal } from '~/ui/components/Snippet';
 
+If you need to extend the capabilities of your application, you can install expo packages or third party react native libraries.
+Sometimes, this is not enough. You may need to write custom native code to tap into obscure platform APIs or use custom Android or iOS dependencies.
+
+Luckily, Expo has first class support for this in the form of Expo Modules.
+
+Expo Modules allow you to write native code in a way that feels natural with minimal boilerplate that is also consistent on both platforms.
```

Comment:
```suggestion
## What are Expo Modules

Expo Modules allow you to write native code in a way that feels natural with minimal boilerplate that is also consistent on both platforms.
```

Reply:
I think the idea was to have a limited introduction in the get started as the overview page seems to be a bit too in detail on the why.